### PR TITLE
feat(file-uploader-2): adding part support of file uploader componen

### PR DESCRIPTION
**Pull request by Tahseen-Ibrahim**
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] My commits have standard messages as mentioned in [Contributing Guidelines](./../blob/next/CONTRIBUTING.md)

### CODE REVIEW

File: packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx [hunk 1]
```diff
@@ -798,20 +801,27 @@ export class FileUploader {
...
                 <span class='highlight'>
                   {TranslationController.t('fileUploader2.retry')}
                 </span>
               </div>
             )}
             {this.description && (
-              <div class='file-uploader__body__dropzone__center__clickable__description'>
+              <div
+                class='file-uploader__body__dropzone__center__clickable__description'
+                part='fw-file-uploader-desc'
```
Comment (arvindanta) [inline packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx:823]: Please add the details of the `parts` in the documentation as well. Part Name and its description

**Review by arvindanta — CHANGES_REQUESTED** [review]

File: packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx (hunk 1 above)
Comment (Tahseen-Ibrahim) [reply packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx:823]: done

Comment (arvindanta) [reply packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx:823]: @Tahseen-Ibrahim Am unable to see any changes in the documentation. Can you please check if you have pushed the changes.

**Review by arvindanta — APPROVED** [review]

File: packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx (hunk 1 above)
Comment (Tahseen-Ibrahim) [reply packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx:823]: yes pushed

**Pull request merged**